3.0/README.md

```diff
@@ -0,0 +1,171 @@
+# TileJSON 3.0
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+**Table of contents**
+
+1. [Purpose](#1-purpose)
+1. [File format](#2-file-format)
+1. [Structure](#3-structure)
+   1. [attribution](#31-attribution)
+   1. [bounds](#32-bounds)
+   1. [center](#33-center)
+   1. [data](#34-data)
+   1. [description](#35-description)
+   1. [grids](#36-grids)
+   1. [legend](#37-legend)
+   1. [maxzoom](#38-maxzoom)
+   1. [minzoom](#39-minzoom)
+   1. [name](#310-name)
+   1. [scheme](#311-scheme)
+   1. [template](#312-template)
+   1. [tilejson](#313-tilejson)
+   1. [tiles](#314-tiles)
+   1. [vector_layers](#315-vector_layers)
+   1. [version](#316-version)
+1. [Examples](#4-examples)
+
+# 1. Purpose
+
+This specification attempts to create a standard for representing metadata about multiple types of web-based map layers, to aid clients in configuration and browsing.
+
+# 2. File Format
+
+TileJSON manifest files use the JSON format as described in [RFC 4627](https://www.ietf.org/rfc/rfc4627.txt).
+
+# 3. Structure
+
+Implementations MUST treat unknown keys as if they weren't present. However, implementations MUST expose unknown key/values in their API so that API users can optionally handle these keys. Implementations MUST treat invalid values for keys as if they weren’t present. If the the field is an optional field and the value is invalid, the default value MAY be applied. If the key is required, implementations MUST treat the entire TileJSON manifest file as invalid and refuse operation.
```

Comment:
Take an example implementation stitches tiles into an image based on a tilejson (and bounding box, scale, zoom). It might not expose *any* key/values, including known ones, so could never be compliant.

Reply:
@pnorman I added the above quote to this PR based on your initial comment. Please provide example wording if you’d like further edits.